### PR TITLE
New Featured Artist: Culprate newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
This is waiting on a few PRs our side before it can go out, but reviews/sanity checks are very welcome.

---

Welcome to osu!'s newest Featured Artist!

### CODE REVIEW

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Featured Artist: Culprate"
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom the osu! Featured Artist lineup — Culprate!
```
Comment (nekodex) [inline news/2018-11-21-new-featured-artist-culprate.md]: I think you accidentally a word

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 2]
```diff
@@ -4,15 +4,13 @@ title: "New Featured Artist: Culprate"
...
 
-Having participated in one of **Culprate**'s grand collaborations with his Twitch community, [Dictate](https://osu.ppy.sh/beatmaps/artists/21) loaned us his voice and recommendation to strike up a deal, and after a while of back and forth, we're finally here today.
-
-We're proud to present to you all a grand selection of **Culprate**'s work — namely a few key tracks from his cult classic album, *Deliverance*, and a much broader list of tracks from the collaborative masterpiece that are the *Unity Project* albums.
+One of **Culprate**'s most recent projects features a massive multi-work collaboration drawn from his Twitch community, of which one of our Featured Artists (namely [Dictate](https://osu.ppy.sh/beatmaps/artists/21)) was a part of. Aptly titled as the *Unity Project*, we are proud to announce that a selection from this body of work and from **Culprate**'s cult classic album *Deliverance* are now available.
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: We are proud to announce that a selection from this body of work, aptly titled the *Unity Project*, as well as **Culprate**'s cult classic album *Deliverance* are now available.

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
+
+One of **Culprate**'s most recent projects features a massive multi-work collaboration drawn from his Twitch community, of which one of our Featured Artists (namely [Dictate](https://osu.ppy.sh/beatmaps/artists/21)) was a part of. Aptly titled as the *Unity Project*, we are proud to announce that a selection from this body of work and from **Culprate**'s cult classic album *Deliverance* are now available.
```
Comment (nekodex) [inline news/2018-11-21-new-featured-artist-culprate.md]: `...features a massive multi-work collaboration...` -> `...is a massive multi-work collaboration...`?

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+The Mappers' Guild has been hard at work over the past few months drawing up some incredible maps to help inspire both players and mappers alike with **Culprate**'s sound, check them out — completing all six will nab you the 4th special Mapper's Guild medal! Collecting them all will make you the envy of your friends. Probably.
+
+*Clicking on the image below will start a download for the pack of all six maps mentioned!*
+
+[![](/wiki/shared/news/2018-11-21-new-featured-artist-culprate/listing.jpg)](https://assets.ppy.sh/beatmap-packs/MappersGuildCulprate.zip)
+
+Here's a sample of what's we've got in store for you to map to:
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: "what's we've"

oh eph honey

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+*This track was featured as the pick for Aspire 2018.*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/624.mp3" type="audio/mpeg">
+</audio>
+
+### Relucent (featuring ZES)
+
+*A special bonus track on the end of the digital edition of Deliverance, Relucent is a celebration of all the acoustic-electronic fusion displayed throughout the album.*
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: on the end -> at the end

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/631.mp3" type="audio/mpeg">
+</audio>
+
+### Impulse (featuring Au5)
+
+*From the second part of the Unity Project albums is perhaps one of the most iconic osu! tracks in recent years, thanks to [handsome's award winning map](https://osu.ppy.sh/beatmapsets/705788#osu/1492654).*
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: From the second part of the Unity Project albums, Impulse is perhaps...

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/625.mp3" type="audio/mpeg">
+</audio>
+
+### Pencilina (featuring Dictate)
+
+*From part two of the Unity Project is a collaboration track with our own Dictate, merging their two distinct sounds to produce something very, very special.*
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: This sentence is basically the same as the previous but changing "second part" to "part two", maybe rewrite?

Comment (Toybickler) [reply news/2018-11-21-new-featured-artist-culprate.md]: From part two..., this collaboration track with our own Dictate merges their two distinct sounds to produce something very, very special.

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+*Rocking in on the third part of the Unity Project albums is a distinctly funky twist on Culprate's classic sound. You may be familiar with this one from the recent OWC 2018 streams.*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/626.mp3" type="audio/mpeg">
+</audio>
+
+### Mechanic Heartbeat (featuring Keota & Sophie Meiers)
+
+*Otherworldly vocals set amid resonating synths and a distinct, two-tap beat pose a unique challenge for mappers to tackle in perhaps one of the most difficult song picks we've selected for the Featured Artist project.*
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: unecessary comma after distinct

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+layout: post
+title: "New Featured Artist: Culprate"
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom to the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
```
Comment (osuShiro) [inline news/2018-11-21-new-featured-artist-culprate.md]: fell in love with

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+    <source src="https://assets.ppy.sh/artists/29/previews/630.mp3" type="audio/mpeg">
+</audio>
+
+--------
+
+Check out all these and more, over at [**Culprate**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/29). If you like what you hear, there's a plethora of links on his listing to all of his social media outlets.
+
+As always, all Featured Artists tracks come pre-timed and are 100% licensed for use in osu! — so worry less, and map more!
+
+We're excited to see what people come up with to his incredible sound, and we're also stoked to see what the entrants of Aspire 2018 have produced with *Acid Rain* sometime during this coming December.
```
Comment (Toybickler) [inline news/2018-11-21-new-featured-artist-culprate.md]: We're excited to see how people reflect his incredible sound...

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom to the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
+
+One of **Culprate**'s most recent projects is a massive multi-work collaboration drawn from his Twitch community, of which one of our Featured Artists (namely [Dictate](https://osu.ppy.sh/beatmaps/artists/21)) was a part of. Aptly titled as the *Unity Project*, we are proud to announce that a selection from this body of work and from **Culprate**'s cult classic album *Deliverance* are now available.
```
Comment (osuShiro) [inline news/2018-11-21-new-featured-artist-culprate.md]: "of which one of our featured artists was a part of" is really heavy and uncomfortable to read. Try to reword it?

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+The Mappers' Guild has been hard at work over the past few months drawing up some incredible maps to help inspire both players and mappers alike with **Culprate**'s sound, check them out — completing all six will nab you the 4th special Mapper's Guild medal! Collecting them all will make you the envy of your friends. Probably.
```
Comment (osuShiro) [inline news/2018-11-21-new-featured-artist-culprate.md]: "check them out" should probably be the start of a new sentence

File: news/2018-11-21-new-featured-artist-culprate.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+*Clicking on the image below will start a download for the pack of all six maps mentioned!*
+
+[![](/wiki/shared/news/2018-11-21-new-featured-artist-culprate/listing.jpg)](https://assets.ppy.sh/beatmap-packs/MappersGuildCulprate.zip)
+
+Here's a sample of what's we've got in store for you to map to:
```
Comment (osuShiro) [inline news/2018-11-21-new-featured-artist-culprate.md]: what's we've got :P

File: news/2018-11-21-new-featured-artist-culprate.md (hunk 9 above)
Comment (Toybickler) [reply news/2018-11-21-new-featured-artist-culprate.md]: it's right, Eph and Culprate fell in love while listening to his music

**Pull request merged**